css/index.css

```diff
@@ -143,4 +221,174 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* About page */
+.about-header {
+    padding-bottom: 20px;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed #1d1d1d;
+}
+
+.about-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.about-block {
+    width: 388px;
+    border-bottom: 1px dashed #1d1d1d;
+    margin-bottom: 20px;
+    padding: 10px; 
+}
+
+.about-block h2 {
+    padding-top: 10px;
+}
+
+.about-block button {
+    font-size: 16px;
+    margin: 15px 0px;
+    background: white;
+    border: 2px solid #1d1d1d;
+    border-radius: 8px;
+    padding: 8px 30px;
+}
+
+.about-footer {
+    padding-bottom: 20px;
+    margin: 20px 0px;
+}
+
+/* Box colors */
+
+#box1 {
+    background: teal;
+}
+
+#box2 {
+    background: gold;
+}
+
+#box3 {
+    background: cadetblue;
+}
+
+#box4 {
+    background: coral;
+}
+
+#box5 {
+    background: crimson;
+}
+
+#box6 {
+    background: forestgreen;
+}
+
+#box7 {
+    background: darkorchid;
+}
+
+#box8 {
+    background: hotpink;
+}
+
+#box9 {
+    background: indigo;
+}
+
+#box10 {
+    background: dodgerblue;
+}
+
+#box11 {
+    background: dodgerblue;
+}
+
+#box12 {
+    background: indigo;
+}
+
+#box13 {
+    background: hotpink;
+}
+
+#box14 {
+    background: darkorchid;
+}
+
+#box15 {
+    background: forestgreen;
+}
+
+#box16 {
+    background: crimson;
+}
+
+#box17 {
+    background: coral;
+}
+
+#box18 {
+    background: cadetblue;
+}
+
+#box19 {
+    background: gold;
+}
+
+#box20 {
+    background: teal;
+}
```

Comment:
`#box1` already had `background: teal;` so you could have reused it (if it were a class)